Document Nested Stack support (#38)
DataDog/datadog-cdk-constructs: README.md

});
```

### Nested Stacks

> Re-add the Datadog CDK Construct to each stack you wish to instrument with Datadog. For example below we initialize the Datadog CDK Construct and call `addLambdaFunctions()` in the `RootStack` and `NestedStack`.

Reviewer: ```suggestion
Add the Datadog CDK Construct to each stack you wish to instrument with Datadog. In the example below, we initialize the Datadog CDK Construct and call `addLambdaFunctions()` in both the `RootStack` and `NestedStack`.
```
